File: Programar em Java/22-Argumentos.java
// YouTube: https://youtu.be/k76K3UraNhU
// Publicação: https://caffeinealgorithm.com/blog/argumentos-em-java/

package com.caffeinealgorithm.programaremjava;

public class Argumentos {
  public void Run() {
    dadosPessoais("Nelson Silva", 25, "Portuguesa");
    dadosPessoais("Larissa Fernandes", 37, "Brasileira");
  }

  public void dadosPessoais(String nome, int idade, String nacionalidade) {
    System.out.printf("Nome: %s\n", nome);
    System.out.printf("Idade: %d\n", idade);
    System.out.printf("Nacionalidade: %s", nacionalidade);
  }
}

/*
  Nome: Nelson Silva
  Idade: 25
  Nacionalidade: Portuguesa
  Nome: Larissa Fernandes
  Idade: 37
  Nacionalidade: Brasileira
*/

File: Programar em C#/22-Argumentos.cs
// YouTube: https://youtu.be/mrbE0tQqbtQ
// Publicação: https://caffeinealgorithm.com/blog/argumentos-em-csharp/

using System;

namespace Base {
  class Argumentos {
    public void Run() {
      DadosPessoais("Nelson Silva", 25, "Portuguesa");
      DadosPessoais("Larissa Fernandes", 37, "Brasileira");
    }

    public void DadosPessoais(string nome, int idade, string nacionalidade) {
      Console.WriteLine($"Nome: {nome}");
      Console.WriteLine($"Idade: {idade}");
      Console.WriteLine($"Nacionalidade: {nacionalidade}");
    }
  }
}

/*
  Nome: Nelson Silva
  Idade: 25
  Nacionalidade: Portuguesa
  Nome: Larissa Fernandes
  Idade: 37
  Nacionalidade: Brasileira
*/
